releasenotes/notes/fix-memory-commutation-checker-dbb441de68706b6f.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 fixes:
   - |
-    Fixed an issue with the :class:`~.CommutationChecker` class where it would
+    Fixed an issue with the :class:`.CommutationChecker` class where it would
```

Comment:
The link to this class isn't working in the rendered release notes, even with this change. Does anyone know why that might be?

Reply:
Looking at: https://github.com/Qiskit/qiskit-terra/blob/992d74f1c3b8230a5a1779bbeb27cb6f142977a2/qiskit/circuit/__init__.py `CommutationChecker` isn't included in an autosummary directive in the module docstring so the docs aren't getting built and included. We can fix that now by adding it to the module documentation like the other classes.

Reply:
Thanks, will do!

Reply:
Fixed in [55d24ff](https://github.com/Qiskit/qiskit-terra/pull/9817/commits/55d24ff738d245d191e21f202c9668564c6903da)